### DISCUSSION TITLE
[FR] Spotify - Support for Playlist-Folders

### DISCUSSION THREAD

**Discussion by Michi0606** [music-assistant/support#1537] [Feature requests and ideas]
This is a feautre request to support Spotify playlist folders. You can use Folders to manage your playlists within spotify, these folder structure could show up in music assistant in "Browse -> Spotify -> Playlists" (URL: "/browse?path=spotify://playlists") or even in playlist-view (URL "/playlists").

best regards Michael

**Comment by DutchJaFO** [comment]
I'd love this as a generic concept for the playlist-view, not just the Spotify playlists.

It would be even better if we could have some kind of meta-data tag attached to playlists so these could be shown in dedicated views once that gets implemented. Although I'd suspect that would require some kind of analysis as I doubt this kind of metadata is available at all.
